plugins/modules/mso_schema_site_anp_epg_domain.py

```diff
@@ -392,8 +451,14 @@ def main():
         deploymentImmediacy=deployment_immediacy,  # keeping for backworths compatibility
         deployImmediacy=deployment_immediacy,  # rename of deploymentImmediacy
         resolutionImmediacy=resolution_immediacy,
+        bindingType=binding_type,
+        netflowPref=netflow_pref,
+        allowPromiscuous=allow_promiscuous,
+        forgedTransmits=forged_transmits,
+        macChanges=mac_changes
     )
 
+
```

Comment:
not sure why additional white line is added, can be removed?

Reply:
I removed it in commit